tests/drivers/sensor/sbs_gauge/src/main.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2021 Leica Geosystems AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+
+#include "test_smartbattery.h"
+
+void test_main(void)
+{
+	ztest_test_suite(framework_tests,
+		ztest_unit_test(test_smartbattery_init),
+		ztest_unit_test(test_smartbattery_getGaugeVoltage),
```

Comment:
Please fix style and use shorter identifiers e.g. `test_sbs_gauge_get_voltage`

Reply:
shorter identifier used

Reply:
This is not addressed, please fix style, do not use camelCase

Reply:
I'm sorry, I missed to change the camelCase style